website/client/components/tasks/column.vue

```diff
@@ -339,45 +310,49 @@ export default {
       selectedItemToBuy: {},
     };
   },
+  created () {
+    // Set Task Column Label
+    this.typeLabel = getTypeLabel(this.type);
+    // Get Category Filter Labels
+    this.typeFilters = getFilterLabels(this.type);
+    // Set default filter for task column
+    this.activateFilter(this.type);
+  },
   computed: {
     ...mapState({
-      tasks: 'tasks.data',
       user: 'user.data',
-      userPreferences: 'user.data.preferences',
     }),
-    onUserPage () {
-      let onUserPage = Boolean(this.taskList.length) && (!this.taskListOverride || this.taskListOverride.length === 0);
+    ...mapGetters({
+      getFilteredTaskList: 'tasks:getFilteredTaskList',
+      getUnfilteredTaskList: 'tasks:getUnfilteredTaskList',
+      getUserPreferences: 'user:preferences',
+      getUserBuffs: 'user:buffs',
+    }),
+    // Temporarily commented till definitely known if onUserPage() is needed or not
```

Comment:
Looks like it's never used (or was before this PR) so we can remove this

Reply:
I want to remove this bit of code, but wasn't sure if it was being used anywhere else. If it isn't I'll take it out.

Reply:
well, it's a computed property, isn't it? I just searched if it was used anywhere in the component but let me know if I'm missing something

Reply:
It is neither used in the same component nor in the direct parent.

onUserPage() resets the dailies filter to default and rewards filter to empty array if the user is not on their own page. Not sure if this type of functionality was designed.

Reply:
cc @TheHollidayInn